Documentation-Requirements.md

```diff
@@ -79,3 +80,21 @@ utility.
       ```
 
   ```
+
+# Images
+
+All binary images files must be in a standard and well supported format such
+as PNG. This format is preferred for vector graphics, such as diagrams, as it
+can store the information more efficiently leading to smaller file sizes. JPEG
+format images are acceptable but are more appropriate for storing photographic
+images.
+
+Where possible, images should be generated using freely available software.
```

Comment:
Suggested rewrite to stay active: When possible, generate images using freely available software.